libraries/AP_Scripting/docs/docs.lua

```diff
@@ -340,6 +340,307 @@ function efi:get_state() end
 ---@return AP_EFI_Backend_ud
 function efi:get_backend(instance) end
 
+-- desc
+---@class airspeed
+airspeed = {}
+
+-- pass in external airspeed data
+---@param param1 airspeed_data_message_t_ud
+function airspeed:handle_external(param1) end
+
+
+-- desc
+---@class ExternalAHRS
+ExternalAHRS = {}
+
+-- desc
+---@param param1 EAHRS_State_ud
+---@param param2 nav_filter_status_flags_t_ud
+---@return boolean
+function ExternalAHRS:handle_scripting(param1, param2) end
+
+-- desc
+---@class mag_data_message_t_ud
+local mag_data_message_t_ud = {}
+
+---@return mag_data_message_t_ud
+function mag_data_message_t() end
+
+-- set field
+---@param value Vector3f_ud
+function mag_data_message_t_ud:field(value) end
+
+-- desc
+---@class baro_data_message_t_ud
+local baro_data_message_t_ud = {}
+
+---@return baro_data_message_t_ud
+function baro_data_message_t() end
+
+-- set field
+---@param value number
+function baro_data_message_t_ud:temperature(value) end
+
+-- set field
+---@param value number
+function baro_data_message_t_ud:pressure_pa(value) end
+
+-- set field
+---@param value integer
+function baro_data_message_t_ud:instance(value) end
+
+-- desc
+---@class airspeed_data_message_t_ud
+local airspeed_data_message_t_ud = {}
+
+---@return airspeed_data_message_t_ud
+function airspeed_data_message_t() end
+
+-- set field
+---@param value number
+function airspeed_data_message_t_ud:temperature(value) end
+
+-- set field
+---@param value number
+function airspeed_data_message_t_ud:differential_pressure(value) end
+
+-- desc
+---@class gps_data_message_t_ud
+local gps_data_message_t_ud = {}
+
+---@return gps_data_message_t_ud
+function gps_data_message_t() end
+
+-- set field
+---@param value number
+function gps_data_message_t_ud:ned_vel_down(value) end
+
+-- set field
+---@param value number
+function gps_data_message_t_ud:ned_vel_east(value) end
```

Comment:
Is this m/s or cm/s?

Reply:
we are always m/s unless stated otherwise

Reply:
Anwered in dev call - m/s for all the bindings.